src/elm/App.elm

```diff
@@ -29,7 +29,7 @@ type Message =
 type Status =
   Init
   | Fetching
-  | Fetched
+  | Fetched String
```

Comment:
This shouldn't be a string, it should be it's own type (e.g. `type UserAction = Enter | Leave`)